src/IssueViz/Models/AnalysisIssueVisualization.cs

```diff
@@ -40,5 +41,9 @@ public AnalysisIssueVisualization(IReadOnlyList<IAnalysisIssueFlowVisualization>
 
         public IReadOnlyList<IAnalysisIssueFlowVisualization> Flows { get; }
         public IAnalysisIssue Issue { get; }
+        public int StepNumber => 0;
+        public bool IsNavigable { get; set; }
```

Comment:
The location viz class now implements `INotifyPropertyChanged`. We don't currently need it for this class but that might change.